bitnami/mysql/README.md

```diff
@@ -46,222 +46,274 @@ The command removes all the Kubernetes components associated with the chart and
-| `initdbScripts`            | Dictionary of initdb scripts                                                                                                                                                                                                                                            | `nil`                                                   |
-| `initdbScriptsConfigMap`   | ConfigMap with the initdb scripts (Note: Overrides `initdbScripts`)                                                                                                                                                                                                     | `nil`                                                   |
+| Name                       | Description                                                                                                                                                                         | Value                  |
+| -------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------- |
+| `image.registry`           | MySQL image registry                                                                                                                                                                | `docker.io`            |
+| `image.repository`         | MySQL image repository                                                                                                                                                              | `bitnami/mysql`        |
+| `image.tag`                | MySQL image tag (immutable tags are recommended)                                                                                                                                    | `8.0.25-debian-10-r37` |
+| `image.pullPolicy`         | MySQL image pull policy                                                                                                                                                             | `IfNotPresent`         |
+| `image.pullSecrets`        | Specify docker-registry secret names as an array                                                                                                                                    | `[]`                   |
+| `image.debug`              | Specify if debug logs should be enabled                                                                                                                                             | `false`                |
+| `schedulerName`            | Use an alternate scheduler, e.g. "stork".                                                                                                                                           | `nil`                  |
```

Comment:
I think this parameter should be in common parameters and not in mysql common parameter as it is not mysql related.

Reply:
Done in b5a70db!